backend/hct_mis_api/apps/registration_data/admin.py

```diff
@@ -217,28 +207,18 @@ def generate_query_for_all_grievances_tickets(rdi: RegistrationDataImport) -> Q:
     def delete_merged_rdi(self, request: HttpRequest, pk: UUID) -> Optional[HttpResponse]:
         try:
             if request.method == "POST":
-                with transaction.atomic(using="default"), transaction.atomic(using="registration_datahub"):
+                with transaction.atomic:
                     rdi = RegistrationDataImport.objects.get(pk=pk)
                     rdi_name = rdi.name
-                    rdi_datahub = datahub_models.RegistrationDataImportDatahub.objects.get(id=rdi.datahub_id)
-                    datahub_individuals_ids = list(
-                        datahub_models.ImportedIndividual.objects.filter(
-                            registration_data_import=rdi_datahub
-                        ).values_list("id", flat=True)
-                    )
                     individuals_ids = list(
                         Individual.objects.filter(registration_data_import=rdi).values_list("id", flat=True)
                     )
-                    rdi_datahub.delete()
                     GrievanceTicket.objects.filter(
                         RegistrationDataImportAdmin.generate_query_for_all_grievances_tickets(rdi)
                     ).filter(business_area=rdi.business_area).delete()
                     rdi.delete()
                     # remove elastic search records linked to individuals
                     business_area_slug = rdi.business_area.slug
-                    remove_elasticsearch_documents_by_matching_ids(
-                        datahub_individuals_ids, get_imported_individual_doc(business_area_slug)
```

Comment:
shouldn't it be like "pending_individuals_ids, get_individual_doc(business_area_slug) "?? @pkujawa

Reply:
After Datahub Removal, there are no separate objects for `before` and `after` merging RDI. These are just Household/Individuals already after import and merging updates their status to MERGED. So for the MERGED RDI there are no Pending objects anymore.

Reply:
@pkujawa also would be good to amend the description to explain what will be happening (to TP, household, individuals and _tickets_)